app/javascript/components/enrollment/steps/Arrival.js

```diff
@@ -9,7 +9,7 @@ import DateInput from '../../util/DateInput';
 class Arrival extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { ...this.props, current: { ...this.props.currentState }, errors: {}, modified: {} };
+    this.state = { ...this.props, current: { ...this.props.currentState }, errors: {}, modified: {}, isEditMode: window.location.href.includes('edit') };
```

Comment:
same here

Reply:
fixed in https://github.com/SaraAlert/SaraAlert/pull/975/commits/441588e6694ae61c4bd1da5b7da3be4de8fd5aa9